types/webdriverio/index.d.ts

```diff
@@ -178,6 +178,9 @@ declare namespace WebdriverIO {
         setWindowRect?: boolean;
         timeouts?: Timeouts;
         unhandledPromptBehavior?: string;
+
+        // wdio-sauce-service specific
+        build?: string;
```

Comment:
Could you please link to the documentation where build is defined? I can't find any documentation at the https://github.com/webdriverio-boneyard/wdio-sauce-service

Reply:
Also, aren't you omitting the `sauceConnectOpts` property?

Reply:
Thanks, I've added `sacueConnectOpts`.  The documentation isn't explicit about the `build` property, but it was mentioned in this issue: https://github.com/webdriverio-boneyard/wdio-sauce-service/issues/33 which links to this example: https://github.com/webdriverio/webdriverio/blob/master/test/conf/travis-ci.js#L21